scylla-cdc-replicator/src/replication_tests.rs

```diff
@@ -173,7 +173,7 @@ mod tests {
         for log in result.rows.unwrap_or_default() {
             let cdc_row = CDCRow::from_row(log, &schema);
             let time = cdc_row.time.to_timestamp().unwrap().to_unix_nanos();
-            if time > *last_read {
+            if time >= *last_read {
```

Comment:
This fix will also accidentally replay changes from the previous operation.

A CDC log table's key is `("cdc$stream_id", "cdc$time", "cdc$batch_seq_no")`, therefore - assuming the same stream id, which is fine here - you can keep track of the last read (cdc$time, cdc$batch_seq_no) pair instead of just cdc$time and keep the strict inqeuality (`>`).

Reply:
Right